boilerplate/fullstack/remix/app/routes/_index.tsx

```diff
@@ -8,158 +8,142 @@ import { getSessionDetails } from "../lib/superTokensHelpers";
 import { TryRefreshComponent } from "../components/tryRefreshClientComponent";
 import { SessionAuthForRemix } from "../components/sessionAuthForRemix";
 
-interface SessionDataForUI {
-    note: string;
-    userId: string;
-    sessionHandle: string;
-    accessTokenPayload: object;
-}
-
-interface SessionForRemixProps {
-    userId: string;
+interface SessionProps {
+    userId?: string;
     sessionHandle?: string;
-    accessTokenPayload: SessionDataForUI;
+    accessTokenPayload: {
+        userId: string;
+        sessionHandle: string;
+        accessTokenPayload: object;
+    };
```

Comment:
Why do we need userId and sessionHandle in accessTokenPayload if the userId and sessionHandle are already a part of the type? It should just be `accessTokenPayload: object`

Also, remove the optional thing from user id and session handle

Reply:
Simplified the type further. It ended up pretty weird, not exactly sure why.